### PR TITLE
Added playbook for installing the VMware VDDK

### PR DESCRIPTION
**Pull request by curt-matthews**
@A-Beck

### CODE REVIEW

File: playbooks/install-vddk.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+- name: CFME | Install VDDK
+  hosts: all
```
Comment (A-Beck) [inline playbooks/install-vddk.yml:3]: Should hosts be narrowed to the just the appliances? `cfme-appliances:!cfme-databases`

Or should the databases have this installed as well?

File: playbooks/install-vddk.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  gather_facts: True
+  tasks:
+    - name: CFME | Install VDDK | Extract VDDK
+      unarchive:
+        src: "{{ cfme_vddk_path }}"
+        dest: /tmp/
```
Comment (A-Beck) [inline playbooks/install-vddk.yml]: For robustness, I'd use the `tempfile` module  to create a temporary directory, then extract there.

```
- name: CFME | Install VDDK | create temp directory
  tempfile:
    state: directory
    prefix: vmware
  register: tmp_dir

- name: CFME | Install VDDK | Extract VDDK
  unarchive:
    src: "{{ cfme_vddk_path }}"
    dest: "{{ tmp_dir['path'] }}"

```

File: playbooks/install-vddk.yml (hunk 1 above)
Comment (curt-matthews) [reply playbooks/install-vddk.yml:3]: Actually, I debated doing something different there but was undecided.  You truly only need to install this on workers in the vCenter zone with Smart State Analysis enabled.

File: playbooks/install-vddk.yml (hunk 2 above)
Comment (curt-matthews) [reply playbooks/install-vddk.yml]: Just updated to include this change.

File: playbooks/install-vddk.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      register: tmp_dir
+
+    - name: CFME | Install VDDK | Extract VDDK
+      unarchive:
+        src: "{{ cfme_vddk_path }}"
+        dest: "{{ tmp_dir['path'] }}"
+
+    - name: CFME | Install VDDK | Sync Folder to /usr/lib
+      synchronize:
+        src: /tmp/vmware-vix-disklib-distrib/
```
Comment (itewk) [inline playbooks/install-vddk.yml]: fix this

**Review by itewk — DISMISSED** [review]

**Review by itewk — APPROVED** [review]

File: playbooks/install-vddk.yml (hunk 3 above)
Comment (curt-matthews) [reply playbooks/install-vddk.yml]: That is fixed.

**Pull request merged**